packages/openapi-generator/README.md

```diff
@@ -20,38 +20,60 @@ $ npm install @sap-cloud-sdk/openapi-generator
 
 <!-- prettier-ignore-start -->
 <!-- commands -->
-* [`generate-openapi-client --inputDir <inputDirectory> --outputDir <outputDirectory>`](#generate-openapi-client---inputdir-inputdirectory---outputdir-outputdirectory)
+* [`generate-openapi-client --input <input> --outputDir <outputDirectory>`](#generate-openapi-client---input-input---outputdir-outputdirectory)
 
-## `generate-openapi-client --inputDir <inputDirectory> --outputDir <outputDirectory>`
+## `generate-openapi-client --input <input> --outputDir <outputDirectory>`
 
-Generate OpenAPI clients, that use the connectivity features of the SAP Cloud SDK for JavaScript.
+Generate OpenAPI client(s), that use the connectivity features of the SAP Cloud SDK for JavaScript/TypeScript.
 
 ```
 USAGE
-  $ generate-openapi-client --inputDir <inputDirectory> --outputDir <outputDirectory>
+  $ generate-openapi-client --input <input> --outputDir <outputDirectory>
 
 OPTIONS
-  -i, --input=input                            (required) Input directory or file for the OpenAPI service definitions.
-  -o, --outputDir=outputDir                    (required) Output directory for the generated OpenAPI client.
-  --clearOutputDir                             Remove all files in the output directory before generation.
+  -i, --input=<path/to/input>                             (required) Specify the path to the directory or file
+                                                          containing the OpenAPI service definition(s) to generate
+                                                          clients for. Accepts Swagger and OpenAPI definitions as YAML
+                                                          and JSON files. Throws an error if the path does not exist.
 
-  --generateJs                                 By default, the generator will also generate transpiled .js, .js.map,
-                                               .d.ts and .d.ts.map files. When set to false, the generator will only
-                                               generate .ts files.
+  -o, --outputDir=<path/to/output>                        (required) Specify the path to the directory to generate the
+                                                          client(s) in. Each client is generated into a subdirectory
+                                                          within the given output directory. Creates the directory if it
+                                                          does not exist. Customize subdirectory naming through
+                                                          `--serviceMapping`.
```

Comment:
This could be subject to change right? See https://github.com/SAP/cloud-sdk-js/pull/1232

Reply:
Yes, will do this on a separate PR.